refresh.py

```
@@ -0,0 +1,223 @@
+
+# ----------------- Config -----------------
+def load_config():
+    cfg_file = Path("refresh.config.json")
+    if cfg_file.exists():
+        try:
+            with cfg_file.open("r", encoding="utf-8") as f:
+                cfg = json.load(f)
+                CONFIG.update(cfg)
+                validate_config(cfg)
+                console.print("✔ Config loaded")
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)